setup.py

```diff
@@ -44,14 +44,12 @@
     license="BSD",
     zip_safe=False,
     keywords='dateparser',
-    python_requires='>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*',
+    python_requires='!=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*',
```

Comment:
```suggestion
    python_requires='>=3.5',
```

However, as the next month Python 3.5 will be deprecated, I think we will be supporting only Python > 3.6. If that's the case, we will need to remove also the `'Programming Language :: Python :: 3.5',` line.